assets/js/common.js

```diff
@@ -762,6 +762,26 @@ var hivepress = {
 				});
 		});
 
+		// Number
```

Comment:
Please simplify the solution, e.g. there's a more simple JS snippet https://stackoverflow.com/questions/41714763/how-do-i-restrict-e-from-html-number-input You can forbid "e" and "+" explicitly, there's no need for a custom pattern parameter. In the task I meant to set a hard-coded HTML attribute as a pattern but it seems that the number input doesn't have this in HTML5.

The result should be just forbidding "+" and "e" to be entered, you can also add a simple check for these characters in PHP below (instead of a custom pattern parameter), but this may not be required if the sanitization method cuts these characters off anyway, or converts the number (try to enter a number with + or e when these are allowed on the front-end, save changes and refresh the page, check how the number was saved - if it saves + or e, then the back-end changes are also needed instead of just JS).

Reply:
Please exclude characters instead of creating a white list, maybe there's a simple snippet. Also, let me know if this front-end solution also resolves the back-end validation, e.g. what happens if you save the number with an "e" before you added this snippet, is it converted to a regular number after saving?